docs/examples/pytorch/main.py

```diff
@@ -292,8 +292,8 @@ def train(train_loader, model, criterion, optimizer, epoch):
     end = time.time()
 
     for i, data in enumerate(train_loader):
-        input = data[0][0][0]
-        target = data[0][1][0].cuda().long()
+        input = data[0]["data"][0]
+        target = data[0]["label"][0].cuda().long()
```

Comment:
This would not work, because it should be called `labels` and not `label`.

Reply:
Done.